packages/terra-search-field/docs/README.md

```diff
@@ -0,0 +1,17 @@
+# Terra Search Field
+
+A search component with a field that automatically performs a search callback after user input.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-search-field`
+
+## Usage
+
+```jsx
+import React from 'react';
+import SearchField from 'terra-search-field';
+
+<SearchField placeholder="Search" />
```

Comment:
Nit: Add onSearch in example since no default search function is provided by the component.